openapi-codegen/core/src/main/scala/sttp/tapir/codegen/SchemaGenerator.scala

```diff
@@ -54,7 +54,10 @@ object SchemaGenerator {
             false
           ) -> "implicit lazy val anyTapirSchema: sttp.tapir.Schema[io.circe.Json] = sttp.tapir.Schema.any[io.circe.Json]"
         )
-      else throw new NotImplementedError("any not implemented for json libs other than circe")
+      else
+        throw new NotImplementedError(
+          s"any not implemented for json libs other than circe (problematic models: ${schemasWithAny.map(_._1)})"
```

Comment:
This can be triggered by surprising things (e.g. an object containing a property that's an enum without a type) so this helps avoid banging heads on walls or keyboards.